src/objects/core/zcl_abapgit_file_status.clas.abap

```diff
@@ -137,16 +134,29 @@ CLASS zcl_abapgit_file_status DEFINITION
         !it_results TYPE zif_abapgit_definitions=>ty_results_tt
       RAISING
         zcx_abapgit_exception .
+    CLASS-METHODS check_local_remote_consistency
```

Comment:
class-methods -> methods ?

Reply:
I tried to keep static what can be static. That's intentional. It gives better visibility what's in what out. Unless there are 5+ params, then nothing is visible anymore :D

Reply:
e.g. rt_result may be an attribute. And maybe it's not that bad idea. But for example if `remote` is an attribute it will be not clear ay more which method read and which modifies it. So these tables remain as params intentionally. While the package and io_dot are only read, and mostly passed to be reused in lower levels. So I moved them to attrs

Reply:
makes sense